src/templates/React InstantSearch widget/src/shims.d.ts

```diff
@@ -0,0 +1,50 @@
+declare module 'react-instantsearch-dom' {
```

Comment:
maybe using @types/react-instantsearch-dom is an option?

Reply:
didn't think of that!
it works just fine (at least for the code this boilerplate produces)
https://github.com/algolia/create-instantsearch-app/pull/456/commits/c42f82506c991bf8235be3f10f3bc3b7eca36974